text/0000-localizing-api-messages.md

```diff
@@ -0,0 +1,135 @@
+
+- Start Date: 03-29-2022
+- RFC PR: 
+- FOLIO Issue: 
+- Submitter : Radhakrishnan Gopalakrishnan (rgopalakrishnan@ebsco.com)
+- Co-Submitter(s) : Zak Burke (zburke@ebsco.com)
+- Sub Group :
+
+# Localizing API (Backend) Messages
+
+## Summary
+  The purpose of this RFC is to define an approach that allows API developer to provide
+messages in the API response depending on the language preference set in the request. 
+
+
+## Motivation
+
+- Help users understand the information coming from the application backend using a language they understand better
+- To allow FOLIO platform to be used by users across the world 
+- To create a tremendous opportunity for growth that we can never could have achieved within just one country.
+
+### Terminology
+- Static Message - E.g. _This is something you need translate_
+- Static Message with placeholder - _You can ONLY renew {n} times_
+ 
+
+### In Scope Requirements/Use cases
+- Return localized messages based on the value passed in the accept-language header
+- Handle static messages, static messages with placeholders
+
+### Out of Scope Requirements/Use cases
```

Comment:
We already have a decision whether messages should be generated by the back-end module itself or at some other module: https://wiki.folio.org/display/DD/Handle+i18n+where+messages+are+generated
Therefore this decision should be listed as out-of-scope of this RFC.

Reply:
You're right, @julianladisch, this RFC covers a lot of the same territory as the RFC. Since tech leads does not meet any longer, the decision log hasn't been well maintained. For architectural decisions like how to manage i18n, we hope the RFC process can be a way to make such decisions in the future and to document them. 

Part of the goal here is to coalesce the i18n info that is scattered across Jira and the decision log and TC meeting notes, and part of it is to pilot the RFC process with a topic that is (hopefully) relatively uncontroversial and already settled, if not completely fleshed out. If we pushed these decisions -- use the `accept-languages` header, handle static values -- out of scope, both goals would suffer. Thus, I think this should stay, even if it covers some existing territory.